build.gradle.kts

```diff
@@ -0,0 +1,162 @@
+
+plugins {
+    id("java-library")
+
+    kotlin("jvm") version "1.6.20"
+    id("maven-publish")
+    id("signing")
+    id("org.jetbrains.dokka") version "1.6.20"
+    id("jacoco")
+    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.9.0"
+    id("org.jlleitschuh.gradle.ktlint") version "10.3.0"
+
+    // TODO: Configure and use this
+    id("com.diffplug.spotless") version "6.11.0"
+}
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+}
+
+group = "com.amazon.ion"
+version = "1.3.0-SNAPSHOT"
+description = "An immutable in-memory representation of Amazon Ion for Kotlin"
+
+val isCI: Boolean = System.getenv("CI") == "true"
+val githubRepositoryUrl = "https://github.com/amazon-ion/ion-java/"
+val isReleaseVersion: Boolean = !version.toString().endsWith("SNAPSHOT")
+
+dependencies {
+    api("com.amazon.ion:ion-java:[1.4.0,)")
+    compileOnly("com.amazon.ion:ion-java:1.4.0")
```

Comment:
Should these minimum requirements be raised given some of the fixes we've recently made in ion-java?

Reply:
I don't know. Ion Element only uses the reader/writer APIs. I think that as long as we aren't using any new APIs, we could allow users of `ion-element-kotlin` to choose any compatible version of `ion-java`.

We will probably need to revisit this once Ion 1.1 is done since there will probably be new APIs in `ion-java` that `ion-element-kotlin` should leverage.